src/LatteContext/Resolver/ClassLatteContextResolver.php

```diff
@@ -38,6 +39,16 @@ public function getFilters(): array
         return $this->latteContext->filterFinder()->find($this->getClassName());
     }
 
+    protected function getTemplateContext(ReflectionClass $reflectionClass, LatteContext $latteContext): TemplateContext
```

Comment:
where is this used?

Reply:
In our code nowhere. But it would be iseful in cistom resolvers mabar is writing.

Reply:
shouldn't be public?

Reply:
Oh shit it should